B23 Ex03 Ronen 319047718 Ido 315942193/GrageUserInterface.cs

```diff
@@ -71,20 +83,29 @@ private void printMenu()
             Console.WriteLine("5. Refuel a car");
             Console.WriteLine("6. Charge electronic car");
             Console.WriteLine("7. Present car by a license plate");
+            Console.WriteLine("8. Exit");
         }
 
-        private bool validateUserActionInput(string i_UserInput, out int o_userInputInteger)
+        private bool validateUserActionInput(string i_UserInput, out int o_UserInputInteger)
         {
             bool isValid = true;
-            int.TryParse(i_UserInput, out o_userInputInteger);
-            if (!(o_userInputInteger > 0 && o_userInputInteger <= 7))
+            int.TryParse(i_UserInput, out o_UserInputInteger);
+            if (!(o_UserInputInteger > 0 && o_UserInputInteger <= 7))
```

Comment:
choose >= or only >

Reply:
I only changed your var naming, you wrote this code